example/objc/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
Q: Is any of these actually required by RN Brownfield? If so, we should try to remove their usage in the future.

Reply:
Not directly, but this file is required by Apple now, so I think it's a good example to heep it

Reply:
So the requirements for that come from RN directly?

Reply:
So requirement for this file comes directly from Apple, to publish your app you need to specify the Privacy Manifest. React Native generates this file with some cocoapods script. I don't want to add it to the .gitignore as this file should be committed. 

Here is a reference: https://medium.com/@dabhir16/a-guide-to-apples-privacy-manifests-in-2024-c3f476b3fa6f